core/src/main/resources/reference.conf

```diff
@@ -392,7 +392,25 @@ services {
   MetadataService {
     class = "cromwell.services.metadata.impl.MetadataServiceActor"
     # Set this value to "Inf" to turn off metadata summary refresh.  The default value is currently "2 seconds".
+    # FIXME: move
     # metadata-summary-refresh-interval = "Inf"
+    #
+    # config {
+      # For higher scale environments, e.g. many workflows and/or jobs, DB write performance for metadata events
+      # can improved by writing to the database in batches. Increasing this value can dramatically improve overall
+      # performance but will both lead to a higher memory usage as well as increase the risk that metadata events
+      # might not have been persisted in the event of a Cromwell crash.
+      #
+      # For normal usage the default value of 1 (effectively no batching) should be fine but for larger/production
+      # environments we recommend a value of at least 500. There'll be no one size fits all number here so testing
+      # is recommended.
+      # db-batch-size = 1
```

Comment:
I originally did, in retrospect I think that's wrong for most users and dropped it back to 1. I don't think the defaults should be catering to a larger scale deployment and turning that number up does bring along some risk of loss of data in the event of a crash. Most small scale users will never notice the difference with the setting at 1, IMO that should be the default.